extra-scientific/z3/autobuild/beyond

```diff
@@ -0,0 +1,7 @@
+cp -vr ${PKGDIR}/usr/lib/python3.* ${PKGDIR}/usr/lib/python2.7
```

Comment:
Dot `abinfo` throughout custom scripts - this is required by the [Styling Manual's Scripting section](https://wiki.aosc.io/developer/packaging/package-styling-manual/#scripting).